user_guide_src/source/installation/installing_composer.rst

```diff
@@ -39,6 +39,8 @@ The command above will create a "project-root" folder.
 If you omit the "project-root" argument, the command will create an
 "appstarter" folder, which can be renamed as appropriate.
 
+.. note:: CodeIgniter autoloader does not allow special characters that are illegal in filenames on certain operating systems. Only allow characters deemed safe for POSIX portable filenames, plus the forward slash for directory separators. So if you install CodeIgniter under the folder that contains the special characters like ``(``, ``)``, etc., CodeIgniter won't work.
```

Comment:
Can we just list down what are those allowed instead of not allowed? I believe the allowed list is smaller than the not allowed.

Reply:
Added allowed symbols.